src/main/webapp/app/admin/admin.module.ts

```diff
@@ -17,11 +19,11 @@ import {
     JhiMetricsService
 } from './';
 
-
 @NgModule({
     imports: [
         JhipsterRegistrySharedModule,
-        RouterModule.forRoot(adminState, {useHash: true})
+        RouterModule.forRoot(adminState, { useHash: true }),
+        NgbModule.forRoot(),
```

Comment:
why this import, its already imported by `JhipsterRegistrySharedModule` and hence should be available here?